pkg/client/unversioned/services.go

```diff
@@ -82,6 +83,13 @@ func (c *services) Update(svc *api.Service) (result *api.Service, err error) {
 	return
 }
 
+// UpdateStatus takes the name of the service and the new status.  Returns the server's representation of the service, and an error, if it occurs.
```

Comment:
UpdateStatus takes a Service object with the new status and applies it as an update to the existing Service.